app/ai/voice/agents/breeze_buddy/managers/calls.py

```
@@ -488,6 +491,28 @@ async def process_backlog_leads():
+                        configurations_dict = locked_lead.metaData.get("configurations")
+                        if configurations_dict and isinstance(
+                            configurations_dict, dict
+                        ):
+                            try:
+                                template.configurations = ConfigurationModel(
+                                    **configurations_dict
+                                )
+                                logger.info(
+                                    f"Applied playground configuration overrides for greeting synthesis (lead {locked_lead.id})"
+                                )
+                            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)